src/core/coreav.cpp

```diff
@@ -478,6 +477,7 @@ void CoreAV::groupCallCallback(void* tox, int group, int peer, const int16_t* da
     audio.playAudioBuffer(call.peers[peer], data, samples, channels, sample_rate);
 }
 
+>>>>>>> 126b52d99047a42907cab322fca439304124e093
```

Comment:
whoops. everything should be fixed now and rebased. If something else needs to be done let me know